subprojects/common/test-okhttp/src/test/kotlin/com/avito/test/http/MockDispatcherTest.kt

```diff
@@ -54,6 +72,44 @@ internal class MockDispatcherTest {
 
         assertThat(response.getBody()?.readUtf8()).isEqualTo("First registered")
     }
+
+    /**
+     * Run with "Repeat: Until failure" option in IDEA run configuration
+     * see MBS-7636
+     */
+    @Disabled("Used only for manual debugging of concurrency problems on dispatcher internal collections")
+    @Test
+    fun `dispatcher - register and dispatching mocks - accessed concurrently`() {
+        var throwable: Throwable? = null
```

Comment:
Are visibility problems possible?
https://stackoverflow.com/a/42479801/1282707

Reply:
maybe; it gets job done here, and not affecting ci; 
so i decided to not invest more here

Reply:
If it happens, it makes tests useless. They don't see an error.
up to you